qnm_filter/utility.py

```diff
@@ -17,16 +18,18 @@ def parallel_compute(self, M_arr, chi_arr, **kwargs):
         array of the values of remnant mass to calculate the likelihood function for
     chi_arr : array-like
         array of the values of remnant spin to calculate the likelihood function for
+    num_cpu : int
+        integer to be based to Parallel as n_jobs. NOTE: passing a positive integer leads to better performance than -1
```

Comment:
Say the performance differs across machines.

Reply:
Done